src/lib/s3Uploader.test.js

```diff
@@ -70,3 +70,26 @@ describe('s3Uploader', () => {
     await expect(got).toBe(response);
   });
 });
+
+describe('s3Uploader.getPresignedUrl', () => {
+  const Bucket = 'fakeBucket';
+  const Key = 'fakeKey';
+  const fakeError = new Error('fake error');
+  const mockGetURL = jest.spyOn(s3, 'getSignedUrl').mockImplementation(() => 'https://example.com');
+  beforeEach(() => {
+    mockGetURL.mockClear();
+  });
+  it('calls getSignedUrl() with correct parameters', () => {
+    const url = getPresignedURL(Key, Bucket);
+    expect(url).toMatchObject({ url: 'https://example.com', error: null });
+    expect(mockGetURL).toHaveBeenCalled();
+    expect(mockGetURL).toHaveBeenCalledWith('getObject', { Bucket, Key, Expires: 360 });
+  });
+  it('calls getSignedUrl() witcih incorrect parameters', async () => {
```

Comment:
No need to update this now, but maybe put a pin in this typo and update it while you're working on the s3 downloads.